src/resources/postcss/customizer-controls.pcss

```diff
@@ -1,25 +1,39 @@
 /* = Customizer Controls (V2)
 ============================ */
+@import "utilities/_variables-full.pcss";
+@import "utilities/_all.pcss";
```

Comment:
do we need the variables skeleton here too? i'm seeing some spacers in this file.